### PR TITLE
HRIS-98 [Markup] Create Notification Page Markup

### PR DESCRIPTION
**Pull request by JohnRoyCabezas**
## Issue Link
https://framgiaph.backlog.com/view/HRIS-98
## Definition of Done

- [x] Users can view all their notifications in one page

## Notes
[figma](https://www.figma.com/file/w3lR4Elj1q5yZgK3Xw0eIx/HRIS-Wireframe?node-id=2299%3A28057&t=Jsw0hOjgzHwldENz-0)

## Pre-condition

npm i 
run docker

## Expected Output

It should allow the user to view all their notifications in one page

## Screenshots/Recordings

Desktop View
image

Mobile View
image
image

### CODE REVIEW

File: client/src/components/molecules/NotificationPopOver/index.tsx [hunk 1]
```diff
@@ -60,7 +60,7 @@ const NotificationPopover: FC<Props> = ({ className }): JSX.Element => {
                 ))}
               </main>
               <footer className="block bg-amber-500 py-2 text-center text-sm font-semibold text-white">
-                <a href="#">See all notifications</a>
+                <a href="notification-management">See all notifications</a>
```
Comment (jsvelte) [inline client/src/components/molecules/NotificationPopOver/index.tsx:63]: This supposed to 
```suggestion
                <Link href="notification-management">See all notifications</Link>
```

File: client/src/components/molecules/NotificationTable/DesktopTable.tsx [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+import Avatar from '~/components/atoms/Avatar'
+import { IDTRNotificationManagement } from '~/utils/interfaces'
+import LineSkeleton from '~/components/atoms/Skeletons/LineSkeleton'
+
+type Props = {
+  table: Table<IDTRNotificationManagement>
+  isLoading: boolean
+}
+
+const DesktopDisclose: FC<Props> = ({ table, isLoading }): JSX.Element => {
```
Comment (jsvelte) [inline client/src/components/molecules/NotificationTable/DesktopTable.tsx:16]:
```suggestion
const NotificationList: FC<Props> = ({ table, isLoading }): JSX.Element => {
```

File: client/src/components/molecules/NotificationTable/DesktopTable.tsx [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+                                <p>
+                                  <span className="font-semibold">Date Filed: </span>
+                                  {row.original.dateFiled}
+                                </p>
+                                <p className="flex items-center">
+                                  <span className="font-semibold">Status: </span>
+                                  <div
+                                    className={classNames(
+                                      'py-0.25  ml-1 rounded-full border  px-1.5',
+                                      row.original.status === 'Pending' &&
```
Comment (jsvelte) [inline client/src/components/molecules/NotificationTable/DesktopTable.tsx:105]: Remove this condition this is for the My DTR and DTR Management...

File: client/src/components/molecules/NotificationTable/DesktopTable.tsx [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+        <div className="flex flex-col px-4 py-3">
+          {Array.from({ length: 30 }, (_, i) => (
+            <LineSkeleton key={i} className="py-1" />
+          ))}
+        </div>
+      ) : (
+        <>
+          {table.getPageCount() === 0 ? (
+            <div className="h-[50vh]">
+              <DiscloseMessage message="No Available Data" />
```
Comment (jsvelte) [inline client/src/components/molecules/NotificationTable/DesktopTable.tsx:29]: No Notification available

**Comment by jsvelte** [comment]
#### Please also fix this dropdown deformation
image

File: client/src/components/molecules/NotificationTable/DesktopTable.tsx [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+              <DiscloseMessage message="Something went wrong" type="error" />
+            </div>
+          ) : (
+            <>
+              {table.getRowModel().rows.map((row) => (
+                <Disclosure key={row.id}>
+                  {({ open }) => (
+                    <>
+                      <Disclosure.Button
```
Comment (jsvelte) [inline client/src/components/molecules/NotificationTable/DesktopTable.tsx:38]: Please used one disclose component that works on desktop and mobile view... no need to create another MobileDisclose component all you have to is to fix the responsiveness and alignment

File: client/src/utils/interfaces/index.tsx [hunk 6]
```diff
@@ -35,3 +35,15 @@ export interface IListOfLeave {
   dateFiled: string
   reason: string
 }
+
+export interface IDTRNotificationManagement {
```
Comment (jsvelte) [inline client/src/utils/interfaces/index.tsx:39]:
```suggestion
export interface INotification {
```

File: client/src/utils/constants/dummyNotificationData.ts [hunk 7]
```diff
@@ -0,0 +1,70 @@
+import { IDTRNotificationManagement } from '../interfaces'
+
+export const dtrNotificationManagement: IDTRNotificationManagement[] = [
```
Comment (jsvelte) [inline client/src/utils/constants/dummyNotificationData.ts:3]:
```suggestion
export const dummyNotificationData: INotification[] = [
```

**Comment by jsvelte** [comment]
Please fix this also this search functionality deformation
image

**Review by jsvelte — APPROVED** [review]
# Approved 💯

**Review by arden-dave — APPROVED** [review]
Approved 👍

**Review by abduljalilpalala — APPROVED** [review]
### Approved! ✅

**Pull request merged**